.env.example

```
@@ -42,6 +42,7 @@ LMNT_API_KEY=
 FLUXIONS_API_KEY=
 DEEPDUB_API_KEY=
 MURFAI_API_KEY=
+HAKIMAI_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HAKIMAI_API_KEY key should go before the HUME_API_KEY key

(UnorderedKey)